git_machete/tests/functional/test_machete.py

```diff
@@ -2361,6 +2361,42 @@ def test_checkout_prs(self) -> None:
               o-chore/comments *  PR #24 (github_user)
             """
         )
+
```

Comment:
While I remember... let's make the naming of the tests for `github` command consistent, i.e. include `..._github_...` part. Currently we've got `test_checkout_prs` but also `test_github_create_pr`